apps/web/components/eventtype/EventAppsTab.tsx

```diff
@@ -64,9 +65,11 @@ export const EventAppsTab = ({ eventType }: { eventType: EventType }) => {
 
   const { shouldLockDisableProps, isManagedEventType, isChildrenManagedEventType } = useLockedFieldsManager(
     eventType,
-    t("locked_fields_admin_description"),
-    t("locked_fields_member_description")
+    methods,
```

Comment:
We now need form methods in order to set discretely the different fields in the event type metadata